crypto/curve25519/curve25519.c

```diff
@@ -4543,8 +4543,6 @@ void GFp_x25519_sc_muladd(uint8_t *s, const uint8_t *a, const uint8_t *b,
 
 /* Prototype to avoid -Wmissing-prototypes warnings. */
 void GFp_ed25519_scalar_mask(uint8_t a[32]);
-void GFp_ed25519_sign(uint8_t *out_sig, const uint8_t *message,
-                      size_t message_len, const uint8_t private_key[64]);
```

Comment:
In this kind of commit, where we replace (C) code with (Rust) code, let's just put the removal of the old code in the same commit as the addition of the new replacement code. That way the poor reviewer doesn't have to wonder if anything is still using the old version.